pandas/tests/frame/test_ufunc.py

```diff
@@ -60,6 +60,22 @@ def test_binary_input_dispatch_binop(dtype):
     tm.assert_frame_equal(result, expected)
 
 
+def test_ufunc_passes_args():
```

Comment:
Does it make sense to parametrize over the functions?

Reply:
Good call, have parameterized

Reply:
subtract maybe too?

Reply:
Sure, have added that